bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -21,7 +17,7 @@ def self.bundler_version(lockfile)
         if (matches = lockfile.content.match(BUNDLER_MAJOR_VERSION_REGEX))
           matches[:version].to_i >= 2 ? V2 : V1
         else
-          FAILOVER
+          DEFAULT
```

Comment:
This is a change over how dependabot currently works. I made it when trying to figure out how to completely remove this file and delegate the logic to RubyGems, but finally gave up so the change is no longer necessary I believe.

I will remove https://github.com/dependabot/dependabot-core/pull/4973/commits/3c1cfcdab1d4165a5341c191736a58b6a087bd39 from this PR, and propose it separately if I feel it's useful.